config/application.rb

```diff
@@ -85,5 +85,11 @@ class Application < Rails::Application
     config.x.recording_unpublish_dir = File.absolute_path(
       ENV.fetch('RECORDING_UNPUBLISH_DIR') { '/var/bigbluebutton/unpublished' }
     )
+
+    # List of BBB server attributes that should not be modified by create API call
+    config.x.create_exclude_params = ENV['CREATE_EXCLUDE_PARAMS']
```

Comment:
Please parse the values into an array here rather than later when the value is used. For example:
```suggestion
    config.x.create_exclude_params = ENV['CREATE_EXCLUDE_PARAMS']&.split(',') || []
```
which will set it to either a list of params or if the environment variable is nil, it'll set an empty list. Same thing for the `join_exclude_params` below.